ToDoApp/ToDoApp/CreateTaskView.swift

```diff
@@ -66,6 +67,12 @@ struct CreateTaskView: View {
                 }
             }
             .foregroundColor(Color(.primaryAccent))
+            .alert(item: $createTaskViewModel.error, content: {error in
```

Comment:
```suggestion
            .alert(item: $createTaskViewModel.error, content: { error in
```
did you remember to run SwiftLint? :)